docs/concepts/nuclio-real-time-functions.ipynb

```diff
@@ -29,7 +29,9 @@
     "```{admonition} Important\n",
     "When working with Nuclio functions (remote / serving / application runtimes) and API Gateways in MLRun, any modifications/deletions should be made in MLRun only. This is also relevant for modifying the spec of a function or API Gateway.\n",
     "<b>Changes that are made in Nuclio are not guaranteed to be synced to MLRun and can cause unexpected behavior.</b> </br>\n",
-    "For example, if you create a remote function in MLRun (which is created in Nuclio) and then delete it in Nuclio, the function status might show as ready, but invoking it will fail.\n",
+    "For example, if you create a remote function in MLRun (which is created in Nuclio) and then delete it in Nuclio, the function status might show as ready, but invoking it will fail.<br>\n",
+    "If you get an error similar to: `Error when translating nuclio names to mlrun names in api gateway: number of functions doesn't match the mlrun functions in annotation`\n",
+    "it means that the API gateway was updated at least once manually in Nuclio itself, without using the MLRun SDK.\n",
```

Comment:
I think we should propose a solution for the problem as well. And the solution in this case is to delete gw and start from scratch